shared/naturalcrit/markdown.js

```diff
@@ -7,6 +7,7 @@ import MarkedExtendedTables     from 'marked-extended-tables';
 import { markedSmartypantsLite as MarkedSmartypantsLite }                                from 'marked-smartypants-lite';
 import { gfmHeadingId as MarkedGFMHeadingId, resetHeadings as MarkedGFMResetHeadingIDs } from 'marked-gfm-heading-id';
 import { markedEmoji as MarkedEmojis }                                                   from 'marked-emoji';
+import MarkedJustifiedParagraphs 														 from 'marked-justified-paragraphs';
```

Comment:
We might rename this extension to "aligned paragraphs" or something. We support left align, right align, and center align. Justified is the one alignment that *isn't* supported.

Reply:
When I worked in print, these were "Right Justified, Left Justified, Center Justified, or Fully ( which we don't use here ) Justified"

It is the proper term. HTML properties name it weird.

Reply:
I can't find a single reference online calling it "right justified, left justified", but we can keep it for now; it's not critical.

When we transfer this over to the naturalcrit org we can re-evaluate. Ideally it would share the same naming conventions as the rest of the Markdown spec just for consistency.

Reply:
Did some digging as well. Can't find anything supporting my memory either. Dunno.

I'll clean it up to modern standards. Stupid hill to die on. :)